src/components/Splashscreen/index.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect } from 'react';
 import SplashScreen from 'react-native-splash-screen';
+import { View } from 'native-base';
```

Comment:
Why from `native-base` and not from `react-native`?

Reply:
meh....auto-import 😞